inspirehep/modules/theme/jinja2filters.py

```diff
@@ -33,6 +33,7 @@
 
 from flask import session, current_app
 from jinja2.filters import evalcontextfilter
+from inspirehep.utils.jinja2 import render_template_to_string
```

Comment:
The problem with big horizontal PRs is that they force a rebase of all other PRs/branches. Hopefully if I bug enough people those imports will start appearing file by file, and then a big horizontal PR is going to be relatively painless : )